src/tactic/ui/container/tab_wdg.py

```diff
@@ -1837,16 +1912,18 @@ def get_tab_header(my, element_name, title, class_name=None, kwargs=None, is_sel
 
 
  
-        #header.add_behavior( {
-        #'type': 'click_up',
-        #'mouse_btn': 'RMB',
-        #'modkeys': '',
-        #'cbjs_action': '''
-        #alert("RMB");
-        #'''
-        #} )
-
         title_div = DivWdg()
+
+        icon = None
+        badge = None
+        if icon:
+            icon = IconWdg(name="whatever", icon=icon)
```

Comment:
can we use a differernt name  like tab_icon.. Both of these if statements are False all the time though